message.go

```diff
@@ -134,8 +135,8 @@ func (this *CommandMsg) pushiOS(server *Server) {
 
 	switch build {
 
-	case "store", "enterprise", "beta":
-		if build == "store" || build == "enterprise" {
+	case "store", "enterprise", "beta", "development":
+		if build == "store" || build == "enterprise" || build == "beta" {
```

Comment:
maybe it would be better to reverse the logic here. check if it's beta first and else do the prod url